otb-tracker/bin

```diff
@@ -13,8 +13,9 @@ shift 2
 export OTB_TRACKER_INTERFACE
 export OTB_TRACKER_CALL
 export OTB_TRACKER_HOST
+export OTB_TRACKER_INITIAL_TIMEOUT
 export OTB_TRACKER_TIMEOUT
-export OTB_TRACKER_RETRY
+export OTB_TRACKER_TRIES
```

Comment:
I used the same as dig. Thought tries was easier to understand than retry because you directly see the maximum number of requests that will be sent